client/dashboard/components/settings/general/store-address.js

```diff
@@ -86,20 +86,21 @@ export function getCountryStateAutofill( options, countryState, setValue ) {
 	useEffect(
 		() => {
 			let filteredOptions = [];
+			const countrySearch = new RegExp( escapeRegExp( autofillCountry ), 'i' );
 			if ( autofillState.length || autofillCountry.length ) {
-				const countrySearch = new RegExp(
-					escapeRegExp( autofillCountry.replace( /\s/g, '' ) ),
-					'i'
-				);
-				filteredOptions = options.filter( option =>
-					countrySearch.test( option.label.replace( '-', '' ).replace( /\s/g, '' ) )
-				);
+				filteredOptions = options.filter( option => countrySearch.test( option.label ) );
 			}
 			if ( autofillCountry.length && autofillState.length ) {
 				const stateSearch = new RegExp( escapeRegExp( autofillState.replace( /\s/g, '' ) ), 'i' );
 				filteredOptions = filteredOptions.filter( option =>
 					stateSearch.test( option.label.replace( '-', '' ).replace( /\s/g, '' ) )
 				);
+
+				if ( filteredOptions.length > 1 ) {
```

Comment:
Nice solution.  Could we also use this same logic to grab state keys like “GA” for “Georgia?”

Reply:
How doeshttps://github.com/woocommerce/woocommerce-admin/pull/3338/commits/9484db968b6b6f63b365d421fca3707090cc562e look?

Reply:
@joshuatf can you 👀 this again?

Reply:
@timmyc I *think* @joshuatf is AFK now, since it's the 11th for him. I believe this comment addresses his feedback though. If things look OK to you, we could merge and do a follow-up PR later if necessary.

Reply:
sounds good to me!